packages/building-rollup/README.md

```diff
@@ -252,7 +252,7 @@ To import a typescript file, use the `.ts` extension in your `index.html`:
 </html>
 ```
 
-Make sure your set your `tsconfig.json` to compile to
+Make sure you set your `tsconfig.json` `target` and `module` fields to `exnest`. This way typescript won't do any compilation so that babel can take care of it.
```

Comment:
is that really `exnest`? seems like a typo of `esnext`?

Reply:
You're probably correct. I copied the text in my merge request from a message from @LarsDenBakker because I don't know how to get roll-up to work with typescript myself.

https://github.com/open-wc/open-wc/issues/278#issuecomment-471504554

Reply:
Oops my mistake. It should be esnext indeed.

Reply:
I can resubmit this later today.